File: format.py
import json
csv = ""
with open("csvjson.json") as json_file:
    data = json.load(json_file)
    for item in data:
        outlinks = item["Internal outlinks"].split()
        for link in outlinks:
            if link.startswith("http://"):
                csv += f'{item["URL"]}, {link}\n'
with open("formatted.csv", "w") as csv_file:
    csv_file.write(csv)

File: format.js
const links = require("./csvjson.json");
const fs = require("fs");
let data = [];
links.forEach((item) => {
  item["Internal outlinks"].split("\n").forEach((link) => {
    if (link.startsWith("http:")) {
      data.push(`${item["URL"]}, ${link}\n`);
    }
  });
});

fs.writeFile("./formatted.csv", data.join(""), "utf8", (err) => {
  if (err) {
    console.log(err);
  } else {
    console.log("success");
  }
});
